app/models/meeting.js

```diff
@@ -33,7 +33,11 @@ export default class Meeting extends Model {
     async: true,
   })
   internalDocumentPublicationActivity;
-
+  @belongsTo('piece', {
+    inverse: 'meeting',
```

Comment:
I see a possible conflict here with https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/e660ce9a47092a46b264d2d824b0b876c51f09c2/app/models/meeting.js#L44

Would rename this (see related app-kaleidos PR). Suggestion: `isMinutesForMeeting`

Reply:
Good catch, I will make the necessary changes